CHANGELOG.md

```diff
@@ -8,6 +8,107 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+
+
+### Known knowns
+
+- Relevant components and interfaces developed considering Gaia-X Trust Framework - 22.10
+- Connection to an external Clearing House service for the entire Catena-X ecosystem is required. The Gaia-X compliance service should generally be capable of recognizing all companies that are able to register with the Catena-X Portal.
+- Conformity to [Asset Administration Shell v3.0](https://industrialdigitaltwin.org/wp-content/uploads/2023/04/IDTA-01002-3-0_SpecificationAssetAdministrationShell_Part2_API.pdf), [API live example](https://app.swaggerhub.com/apis/Plattform_i40/AssetAdministrationShellRegistryServiceSpecification/V3.0_SSP-001#/Asset%20Administration%20Shell%20Registry%20API/GetAssetAdministrationShellDescriptorById) (AAS) is given. The DTR “/query” interface is finally deprecated and no longer available with this Release as announced. Details can be found in the [changelog](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/main/CHANGELOG.md) of the respective component.
+
+- EDC provides no new release compared to 23.12 but only patches: EDC version 0.5.4 is part of 24.03 with the following limitations:
+  Standard Security Tests have been performed, but it is strongly recommended for each EDC operator to apply own security measures. Especially the Dynamic Application Security Testing (DAST) needs to be executed for each specific operation environment and instance.
+  Tractus-X release guidelines (TRGs) have been fulfilled.
+
+- FOSS components developed and pre-tested to TRL6 with artificial test data only
+- No crosscheck of functionality in verification environment
+- No execution of Load-, Performance- or Penetration Tests
+- No assignment of Export Control Classification Numbers (ECCN) to FOSS components
+- GeoBlocking recommended for Operations (GBaaS)
+- Security concept (overall) recommended for Operations
+
+- Listed components in "Unchanged, untested" category were not verified in combination with the current Release package and are only compatible with Catena-X Standards as specified. Use at your own risk at this time.
+
+### Runtime
+
+- Tested on [Kubernetes](https://en.wikipedia.org/wiki/Kubernetes) versions: `1.27.3`
+- Tested with [PostgreSQL](https://en.wikipedia.org/wiki/PostgreSQL) versions: `15`
+  - Three exceptions apply for
+    - Golden Record Business Partner Number (BPN) Service
+    - Managed Identity Wallet (MIW)
+    - Managed Service Orchestrator (MSO)
+
+    Those components violate TRG 5.07 as they still operate on Postgresql DB Version 14.x (not 15), but have been successfully tested for the current Release.
```

Comment:
update wording to
Those components violate TRG 5.07 as they operate on Postgresql DB Version 16 (not 15), but have been successfully tested for the current Release.

Reply:
done